packages/opencensus-core/src/trace/model/span.ts

```diff
@@ -193,9 +193,10 @@ export class Span implements types.Span {
   /**
    * Adds an atribute to the span.
    * @param key Describes the value added.
-   * @param value The result of an operation.
+   * @param value The result of an operation. If the value is a typeof object
+   * it has to be JSON.stringify-able, cannot contain circular dependencies.
```

Comment:
Nit: could you indent "it has..." by 4 spaces to show it is a continuation of the @param JSDoc tag?